lib/command/run-workers.js

```diff
@@ -209,3 +214,12 @@ function simplifyObject(object) {
       return obj;
     }, {});
 }
+
+const updateFinishedTests = (test) => {
+  const { id } = test;
+  if (finishedTests[id]) {
+    const stats = { passes: 0, failures: -1, tests: 0 };
```

Comment:
could you enlighten me why `failures: -1` 😅

Reply:
@PeterNgTr if `finishedTests` already have that particular id, it means that the test is updated again. Now there are 2 possibilities it may be passed or failed which is updated in the event.
So for both cases, failure should be decremented